com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/UnityBridge.kt

```diff
@@ -5,244 +5,192 @@ import com.chartboost.heliumsdk.*
 import com.chartboost.heliumsdk.ad.*
 import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
 import com.chartboost.heliumsdk.domain.ChartboostMediationAdException
-import com.chartboost.mediation.unity.EventProcessor.LoadEventConsumer
-import com.chartboost.mediation.unity.EventProcessor.EventWithErrorConsumer
+import com.chartboost.mediation.unity.AdWrapper.Companion.wrap
 import com.chartboost.mediation.unity.EventProcessor.EventConsumer
+import com.chartboost.mediation.unity.EventProcessor.EventWithErrorConsumer
+import com.chartboost.mediation.unity.EventProcessor.LoadEventConsumer
 import com.chartboost.mediation.unity.EventProcessor.serializeEvent
-import com.chartboost.mediation.unity.EventProcessor.serializeEventWithError
+import com.chartboost.mediation.unity.EventProcessor.serializeEventWithException
 import com.chartboost.mediation.unity.EventProcessor.serializeLoadEvent
-import com.chartboost.mediation.unity.EventProcessor.serializePlacementIlrdData
-import com.chartboost.mediation.unity.AdWrapper.Companion.wrap
 import com.unity3d.player.UnityPlayer
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
 @Suppress("NAME_SHADOWING")
```

Comment:
Probably should not suppress this and should address it whenever it occurs.

Reply:
This was just related to C#/Kotlin APIs naming conventions and so, they are to be removed in the future, but we are already following Kotlin naming standards.